File: phase1/rest/app/Model/Student.php
<?php
class Student extends AppModel {
    public $validate = array(
	'compid' => array(
		'rule' => 'notEmpty'
	),
	'fname' => array(
		'rule' => 'notEmpty'
	),
	'lname' => array(
		'rule' => 'notEmpty'
	),
	'role' => array(
		'rule' => 'notEmpty'
	),
	   
    );
}

File: phase4/src/edu/virginia/cs/cs4720/adamhunterdan/phase2/Student.java
package edu.virginia.cs.cs4720.adamhunterdan.phase2;

public class Student {
	public String compid;
	public String fname;
	public String lname;
	public String role;
	
	public String getCompid() {
		return compid;
	}

	public void setCompid(String compid) {
		this.compid = compid;
	}

	public String getFname() {
		return fname;
	}

	public void setFname(String fname) {
		this.fname = fname;
	}

	public String getLname() {
		return lname;
	}

	public void setLname(String lname) {
		this.lname = lname;
	}

	public String getRole() {
		return role;
	}

	public void setRole(String role) {
		this.role = role;
	}

	@Override
	public String toString() {
		return this.lname + ", " + this.fname + " (" + this.compid + ") - " + this.role; 
	}
}
